lib/reek/ast.rb

```diff
@@ -0,0 +1,6 @@
+# FIXME: Require this somewhere
+module Reek
+  # @api private
+  module AST
```

Comment:
I don't get it, why the empty module? And why require it somewhere when it's empty? Is that a yard-thing?